src/libgdiplus.vcxproj

```diff
@@ -91,6 +91,7 @@
       <PreprocessorDefinitions>HAVE_LIBJPEG;HAVE_LIBTIFF;HAVE_LIBPNG;_WINDLL;WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <!-- FIXME: To align with the GDI+ calling convention, this should be StdCall. Only relevant on x86 -->
       <CallingConvention>Cdecl</CallingConvention>
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
Actually I get this warning for any Debug configuration, be it Win32 or x64

This is not needed for the Release configuration, as it is already set.

I've updated this in a PR feedback commit